src/components/SelectionListWithSections/Search/ExpenseReportListItem.tsx

```diff
@@ -0,0 +1,139 @@
+            currentSearchKey,
+            onDEWModalOpen,
+        );
+    }, [currentSearchHash, reportItem, onSelectRow, snapshotReport, snapshotPolicy, lastPaymentMethod, currentSearchKey, onDEWModalOpen]);
+
+    const handleCheckboxPress = useCallback(() => {
+        onCheckboxPress?.(reportItem as unknown as TItem);
+    }, [onCheckboxPress, reportItem]);
+
+    const listItemPressableStyle = useMemo(
+        () => [
+            styles.selectionListPressableItemWrapper,
+            styles.pv3,
+            styles.ph3,
+            // Removing background style because they are added to the parent OpacityView via animatedHighlightStyle
+            styles.bgTransparent,
+            item.isSelected && styles.activeComponentBG,
+            styles.mh0,
+        ],
+        [styles, item.isSelected],
+    );
+
+    const listItemWrapperStyle = useMemo(
+        () => [
+            styles.flex1,
+            styles.userSelectNone,
+            isLargeScreenWidth ? {...styles.flexRow, ...styles.justifyContentBetween, ...styles.alignItemsCenter} : {...styles.flexColumn, ...styles.alignItemsStretch},
+        ],
+        [styles, isLargeScreenWidth],
+    );
+
+    const animatedHighlightStyle = useAnimatedHighlightStyle({
+        borderRadius: variables.componentBorderRadius,
+        shouldHighlight: item?.shouldAnimateInHighlight ?? false,
+        highlightColor: theme.messageHighlightBG,
+        backgroundColor: theme.highlightBG,
+    });
+
+    return (
+        <BaseListItem
+            item={item}
+            pressableStyle={listItemPressableStyle}
+            wrapperStyle={listItemWrapperStyle}
+            containerStyle={[styles.mb2]}
+            isFocused={isFocused}
+            showTooltip={showTooltip}
+            canSelectMultiple={canSelectMultiple}
+            onSelectRow={onSelectRow}
+            pendingAction={item.pendingAction}
+            keyForList={item.keyForList}
+            onFocus={onFocus}
+            onLongPressRow={onLongPressRow}
+            shouldSyncFocus={shouldSyncFocus}
+            hoverStyle={item.isSelected && styles.activeComponentBG}
+            pressableWrapperStyle={[styles.mh5, animatedHighlightStyle]}
+            shouldShowRightCaret={isLargeScreenWidth}
+            shouldUseDefaultRightHandSideCheckmark={false}
+        >
+            {(hovered) => (
+                <ExpenseReportListItemRow
+                    item={reportItem}
+                    policy={snapshotPolicy}
+                    isActionLoading={isLoading}
+                    showTooltip={showTooltip}
+                    canSelectMultiple={canSelectMultiple}
+                    onCheckboxPress={handleCheckboxPress}
+                    onButtonPress={handleOnButtonPress}
+                    avatarBorderColor={theme.highlightBG}
+                    isSelectAllChecked={!!reportItem.isSelected}
+                    isIndeterminate={false}
+                    isDisabledCheckbox={item.isDisabledCheckbox}
```

Comment:
Can `reportItem` be used here? Sometimes we use `item`, sometimes `reportItem`, which would cause confusion later